src/Data/X509/Extra.hs

```diff
@@ -223,8 +225,13 @@ parseSAN name =
         BL.toStrict $ BS.toLazyByteString (BS.word32BE bytes)
 
     ipv6ToBS :: IPv6 -> ByteString
+#if MIN_VERSION_ip(1,5,0)
```

Comment:
commit is here https://github.com/andrewthad/haskell-ip/commit/0742cd48f5a49a0e8718f6eb559c70d5192ac799
changelog here https://github.com/andrewthad/haskell-ip/blob/master/changelog.md#150---2019-03-23